Raylib-cs/types/Color.cs

```diff
@@ -110,10 +109,9 @@ public Color(float r, float g, float b, float a)
     /// </summary>
     public Color(float r, float g, float b)
     {
-        //   X = (byte)Math.Clamp(MathF.Round(x * 255), 0f, 255f);
-        this.R = (byte)((r < 0) ? 0 : ((r > 1) ? 255 : ((r * 255) + .5f)));
-        this.G = (byte)((g < 0) ? 0 : ((g > 1) ? 255 : ((g * 255) + .5f)));
-        this.B = (byte)((b < 0) ? 0 : ((b > 1) ? 255 : ((b * 255) + .5f)));
+        this.R = (byte)(Math.Clamp(r, 0.0f, 1.0f) * 255);
```

Comment:
Needs testing to make sure the behavior is the same. Also why Math.Clamp vs MathF.Clamp?